cahierCharges.tex

```diff
@@ -37,18 +37,55 @@
 \section{Introduction}
 
 \subsection{Objet du document}
+Ce document décrit tous les services que doit rendre le prototype de
+logiciel de gestion de fuseaux horaires ainsi
+que toutes les exigences que ces services doivent satisfaire.
 
 
 \subsection{Portée du document}
 
+Ce document est destiné à expliciter et formaliser le besoin du client
+ dans le cadre du projet DEV qui se
+déroule pendant le 1er semestre de 2017.
+
+
 
 \subsection{Abréviations}
+\label{sec:abreviation}
+
+\begin{tabular}[c]{|p{1cm}|p{2.6cm}|p{12.4cm}|}
+  \hline
+  \textbf{Abr.} & \textbf{Signification} & \textbf{Libellé}
+  \\\hline
+  \textit{VIT} & \textit{Vitale} & \textit{Exigence fonctionnelle ou non fonctionnelle indispensable}
+  \\\hline
+  \textit{IMP} & \textit{Importante} & \textit{Exigence souhaitée mais non exigée}
+  \\\hline
+  \textit{MIN} & \textit{Mineure} &
+  \textit{Exigence non exigée immédiatement, mais qui devra être prise en compte
+    ultèrieurement par le produit (impact sur l'évolutivité)}
+  \\\hline
+  \textit{CNIL} & \textit{Commission Nationale de l'Informatique et des Libertés} &
+  \textit{Institution indépendante chargée de veiller au respect de l'identité humaine, de la vie privée et des libertées dans le monde numérique en France}
+  \\\hline
+  \textit{GPL} & \textit{Licence Publique Générale GNU} &
+  \textit{Licence qui fixe les conditions légales de distribution des logiciels libres du projet GNU}
+  \\\hline
+\end{tabular}
 
 \section{Les objectifs du produit}
 
 \subsection{Définition du produit}
+Le produit demandé par le client est un prototype pour la gestion d'un
```

Comment:
En l'occurrence un client ne va jamais demander un prototype.
Le client veut un produit finit avec toutes les fonctionnalités dont il a besoin.
C'est votre travail de chef de projet de commencer par lui proposer un prototype et d'itérer dessus jusqu'à atteindre un produit finit.